app/frontend/components/common/footer.js

```diff
@@ -23,19 +33,75 @@ const Footer = () =>
           'View on Github'
         )
       ),
-      '/',
+      ' /',
       h(
         'small',
         {class: 'contact-link'},
-        h('a', {href: 'mailto:cardanolite@vacuumlabs.com'}, 'Contact us')
+        h('a', {href: 'mailto:cardanolite@vacuumlabs.com', target: '_blank'}, 'Contact us')
       ),
       '/',
       h(
         'small',
         {class: 'contact-link'},
-        h('a', {href: 'https://twitter.com/hashtag/cardanolite'}, '#cardanolite')
+        h('a', {href: 'https://twitter.com/hashtag/cardanolite', target: '_blank'}, '#cardanolite')
+      )
+    ),
+    h(
+      'p',
+      undefined,
+      h('div', undefined, 'Donations are really appreciated! '),
+      h(
+        'div',
+        undefined,
+        donateBtcUrl &&
```

Comment:
I would remove this &&, same for remaining coins. I would assume that the urls and addresses are available